src/main/kotlin/com/example/address_project/domain/street_number/domain/StreetNumber.kt

```diff
@@ -46,6 +43,9 @@ class StreetNumber (
     @Column(columnDefinition = "VARCHAR(40)")
     val engLegalName: String? = null,
 
+    @Enumerated(EnumType.STRING)
+    val type: Type? = null,
+
```

Comment:
? 타입이 어떻게 null이 가능한가요???

Reply:
수정하겠습니다

Reply:
type이 필요한 경우는 검색기능에서 필요한게 아닌가요? 
null로 하지 않으면 데이터 파싱하는 부분에도 들어가게 됩니다.

Reply:
어.. 그게 제 말은 null 설정을 안하면 
```suggestion
     siGunGuName = columns[4],
     legalEmdName = columns[5],

```
이런 파싱코드에 들어가게 되서 null로 설정해야 했습니다. 라는 말이였습니다

Reply:
그러니까 저도 파싱 코드에 들어가는 게 맞다고 생각합니다 그게 아니었으면 애초에 type이 엔티티에 추가될 이유가 없습니다

Reply:
아 넵 알겠습니다